### PR TITLE
This is my homework in English

### PR DESCRIPTION
**Pull request by Alexandr-R-94**
This is a vocabulary assignment

### CODE REVIEW

**Review by olgaz-skiba — APPROVED** [review]
please, approve my pullrequest

File: homework/english/raby_alexandr/lection1/vocabulary.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+1. I boot Win10 before work.
```
Comment (DenisAbramovDm) [inline homework/english/raby_alexandr/lection1/vocabulary.md:1]: Booted. If u say about past time

File: homework/english/raby_alexandr/lection1/vocabulary.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+2. I create my first homework in Github.
```
Comment (DenisAbramovDm) [inline homework/english/raby_alexandr/lection1/vocabulary.md:2]: Created

File: homework/english/raby_alexandr/lection1/vocabulary.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+3. Cannot connect to the specified server.
+4. We are debuging their code now.
```
Comment (DenisAbramovDm) [inline homework/english/raby_alexandr/lection1/vocabulary.md:4]: Debugging

File: homework/english/raby_alexandr/lection1/vocabulary.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+1. I boot Win10 before work.
+2. I create my first homework in Github.
+3. Cannot connect to the specified server.
+4. We are debuging their code now. 
+5. She is developing new application at the moment.
+6. He disables his PC before going to bed.
```
Comment (DenisAbramovDm) [inline homework/english/raby_alexandr/lection1/vocabulary.md:6]: In this case u may to add "every day" at the end of sentence

File: homework/english/raby_alexandr/lection1/vocabulary.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+7. I am disconnecting its the network now.
```
Comment (DenisAbramovDm) [inline homework/english/raby_alexandr/lection1/vocabulary.md:7]: Seems something is wrong here, i cant understand  what u wanted to write

File: homework/english/raby_alexandr/lection1/vocabulary.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+8. I am displaying the picture at the moment.
+9. She usually enables  this function.
+10. His chat bot executes different commands.
+11. She constantly emplements new functions in this chat bot.
+12. Are you installing this PC-game today?
+13. Do you help me to restore my program?
```
Comment (DenisAbramovDm) [inline homework/english/raby_alexandr/lection1/vocabulary.md:13]: It is better to use phrase "can u help me". And it is more correct

File: homework/english/raby_alexandr/lection1/vocabulary.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+14. Are you pluiging in my PC to the network todey?
```
Comment (DenisAbramovDm) [inline homework/english/raby_alexandr/lection1/vocabulary.md:14]: This sentence can be written much better

File: homework/english/raby_alexandr/lection1/vocabulary.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+15. I am uninstalling program now.
+16. We need to upgrade our program.
+17. She need verifies all bugs.
```
Comment (DenisAbramovDm) [inline homework/english/raby_alexandr/lection1/vocabulary.md:17]: Something is not good here. I cant say what exactly

File: homework/english/raby_alexandr/lection1/vocabulary.md (hunk 1 above)
Comment (Alexandr-R-94) [reply homework/english/raby_alexandr/lection1/vocabulary.md:1]: We study the Present Simple and Present Continuous. Why use the Past Simple?

File: homework/english/raby_alexandr/lection1/vocabulary.md (hunk 2 above)
Comment (Alexandr-R-94) [reply homework/english/raby_alexandr/lection1/vocabulary.md:2]: We study the Present Simple and Present Continuous. Why use the Past Simple?

File: homework/english/raby_alexandr/lection1/vocabulary.md (hunk 3 above)
Comment (Alexandr-R-94) [reply homework/english/raby_alexandr/lection1/vocabulary.md]: Thank you for your comment

File: homework/english/raby_alexandr/lection1/vocabulary.md (hunk 4 above)
Comment (Alexandr-R-94) [reply homework/english/raby_alexandr/lection1/vocabulary.md]: Perhaps this is appropriate.

File: homework/english/raby_alexandr/lection1/vocabulary.md (hunk 5 above)
Comment (Alexandr-R-94) [reply homework/english/raby_alexandr/lection1/vocabulary.md]: Perhaps you need to write "it's"

File: homework/english/raby_alexandr/lection1/vocabulary.md (hunk 6 above)
Comment (Alexandr-R-94) [reply homework/english/raby_alexandr/lection1/vocabulary.md]: Thank you for your comment

File: homework/english/raby_alexandr/lection1/vocabulary.md (hunk 1 above)
Comment (DenisAbramovDm) [reply homework/english/raby_alexandr/lection1/vocabulary.md:1]: Then u should correctly construct the sentence and add keywords where it is necessary

File: homework/english/raby_alexandr/lection1/vocabulary.md (hunk 2 above)
Comment (DenisAbramovDm) [reply homework/english/raby_alexandr/lection1/vocabulary.md:2]: So maybe u wanted to say  "I am creating"? See the rules more attentively

**Review by DenisAbramovDm — APPROVED** [review]

**Pull request merged**